models/sources/sources.sql

```diff
@@ -10,14 +10,19 @@ select
     cast(null as {{ type_string() }}) as database,
     cast(null as {{ type_string() }}) as schema,
     cast(null as {{ type_string() }}) as source_name,
+    cast(null as {{ type_string() }}) as source_description,
     cast(null as {{ type_string() }}) as loader,
     cast(null as {{ type_string() }}) as name,
     cast(null as {{ type_string() }}) as identifier,
     cast(null as {{ type_string() }}) as loaded_at_field,
     {% if target.type == 'snowflake'%}
-        cast(null as {{ type_array() }}) as freshness
+        cast(null as {{ type_array() }}) as freshness,
+        cast(null as {{ type_array() }}) as source_meta,
+        cast(null as {{ type_array() }}) as meta
     {% else %}
-        cast(null as {{ type_json() }}) as freshness
+        cast(null as {{ type_json() }}) as freshness,
+        cast(null as {{ type_json() }}) as source_meta,
+        cast(null as {{ type_json() }}) as meta   
```

Comment:
```suggestion
```
Please could you move `source_meta` and `meta` below the `endif`? They are always json types.